Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccreissuance/ui/consent/DccReissuanceConsentViewModel.kt

```diff
@@ -83,8 +94,9 @@ class DccReissuanceConsentViewModel @AssistedInject constructor(
     fun openPrivacyScreen() = event.postValue(OpenPrivacyScreen)
 
     internal data class State(
-        val certificate: DccV1.MetaData?,
+        val certificateList: List<DccReissuanceItem>,
```

Comment:
You can just use  List<DccV1.Metadata>, as you only need the certificate for the UI (or did I overlook something?) To keep it independent from the data structure of the wallet